conda/meta.yaml

```diff
@@ -1,22 +1,27 @@
+{% set name = "zstash" %}
+{% set version = "0.4.0" %}
+
 package:
-    name: zstash
-    version: 0.4.0
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-    git_url: https://github.com/E3SM-Project/zstash
-    git_rev: v0.4.0
+  git_url: https://github.com/E3SM-Project/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9fce7561efa0d512567f89219a1f1642210810523a550b1974a17fdba1b6f544
 
 build:
-    script: python setup.py install
+  number: 1
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
 
 requirements:
-  build:
-    - python 3
-    - setuptools
+  host:
+    - python >=3.6
```

Comment:
Here and below, it is more typical to constrain python in this way, though you could also specify that it has to be less than 4 if you have reason to be worried.